odpf/assets/bucket.proto

```diff
@@ -66,19 +65,15 @@ message Blob {
   // Example: `gcs`.
   string source = 3;
 
-  // The description of the blob.
-  // Example: `This is a config file for x app`
-  string description = 4;
-
   // The length of the object content.
   // Example: `300`
-  int64 size = 5;
+  int64 size = 4;
```

Comment:
This will break if this one is currently used. Is it fine?

Reply:
right now nothing is using it and yes it is a breaking changes.

Currently this proto is only being used by Meteor only and still changing quite frequently.
For now we can still afford breaking changes until some other services start using it.

Reply:
noted.